.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
   - DJANGO_VERSION=1.5
```

Comment:
Yeah, that was my first though, although it appears that DRF3 support goes [all the way back to 1.4](https://github.com/tomchristie/django-rest-framework/tree/master#requirements)!  i'm fine either way,